### PR TITLE
Feachure/quick buy

### PR DESCRIPTION
**Pull request by quasisamurai**

### CODE REVIEW

File: contracts/Market.sol [hunk 1]
```diff
@@ -166,7 +166,7 @@ contract Market is Ownable {
 
         require(_benchmarks.length == benchmarksQuantity);
 
-        uint lockedSum = 0;
+        uint lockedSum;
```
[the remark this answers is not in the document]
Comment (quasisamurai) [reply contracts/Market.sol]: due it anyway takes value on next strings

File: contracts/Market.sol [hunk 2]
```diff
@@ -215,6 +215,33 @@ contract Market is Ownable {
         return true;
     }
 
+
+    function QuickBuy(uint askID) public {
+        require(orders[askID].orderType == OrderType.ORDER_ASK);
+        require(orders[askID].orderStatus == OrderStatus.ORDER_ACTIVE);
+
+
+        var ask = orders[askID];
```
[the remark this answers is not in the document]
Comment (quasisamurai) [reply contracts/Market.sol]: Due if ask odrer doesn't exists or incative, we don't neeed to  store the whore deal in EVM memory.

File: contracts/Market.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sokel) [reply contracts/Market.sol]: ok

File: contracts/Market.sol [hunk 3]
```diff
@@ -217,13 +217,11 @@ contract Market is Ownable {
...
-        require(orders[askID].orderType == OrderType.ORDER_ASK);
-        require(orders[askID].orderStatus == OrderStatus.ORDER_ACTIVE);
-
-
         var ask = orders[askID];
+        require(ask.orderType == OrderType.ORDER_ASK);
+        require(ask.orderStatus == OrderStatus.ORDER_ACTIVE);
 
-        require(pr.CheckProfileLevel(msg.sender, uint(ask.identityLevel)));
+        require(pr.CheckProfileLevel(msg.sender, uint(ask.identgtyLevel)));
```
Comment (Sokel) [inline contracts/Market.sol]: miscast?

**Pull request merged**